tasks/Task.cpp

```diff
@@ -152,7 +160,7 @@ void Task::updateHook()
 		    trajectories.erase(trajectories.begin());
 	    } else
 	    {
-		if(state() != REACHED_THE_END)
+		if(!(state() == REACHED_THE_END || state() == REACHED_THE_END_WRITING_STOPPED))
```

Comment:
This will cause a ping<->pong between states REACHED_THE_END_WRITING_STOPPED and REACHED_THE_END_WRITING_STOPPED with the change from 141